Source/IGListCollectionView.m

```diff
@@ -11,25 +11,6 @@
 
 @implementation IGListCollectionView
 
-- (void)commonInit {
-    self.backgroundColor = [UIColor whiteColor];
-    self.alwaysBounceVertical = YES;
-}
-
-- (instancetype)initWithFrame:(CGRect)frame collectionViewLayout:(UICollectionViewLayout *)layout {
-    if (self = [super initWithFrame:frame collectionViewLayout:layout]) {
-        [self commonInit];
```

Comment:
actually, i'm fine either way. 

but i agree that white + bouncing is convenient. 

(also, IG implementations probably rely on these defaults right now?)